src/stuff/Main.js

```diff
@@ -1,27 +1,62 @@
 import { Component } from 'react';
+import Container from 'react-bootstrap/Container'
+import Row from 'react-bootstrap/Row'
+import Col from 'react-bootstrap/Col'
+import Card from 'react-bootstrap/Card'
 
-class Main extends Component{
+export default class Main extends Component{
+  
   render() {
+    
+      var bios = this.props.animalBios;
+
     return (
-    <>
-    <HornedBeast title={'The Beast'} imageURL={'https://upload.wikimedia.org/wikipedia/en/4/4a/Disney-beast.PNG'} description={'He\'s no monster, Gaston. <b>You</b> are!'}/>
-    <HornedBeast title={'Mr. Narwhal'} imageURL={'https://image.pngaaa.com/515/2033515-middle.png'} description={"Bye Buddy! Hope you fine your dad!"}/>
-    </>
+      <Container fluid>
+        <h2>{this.props.message}</h2>
+        <Row>
+          <HornedBeast bio={bios[0]}/>
+          <HornedBeast bio={bios[1]}/>
+          <HornedBeast bio={bios[2]}/>
+        </Row>
+        <Row>
+          <HornedBeast bio={bios[3]}/>
+          <HornedBeast bio={bios[4]}/>
+          <HornedBeast bio={bios[5]}/>
+        </Row>
+
+      </Container>
```

Comment:
Great work here too, today in class you guys covered Map() so I am sure this will change soon :)

For this section, consider cleaning up the spacing here a bit.